lib/rdkafka/admin.rb

```diff
@@ -51,6 +52,16 @@ def create_topic(topic_name, partition_count, replication_factor)
         raise Rdkafka::Config::ConfigError.new(error_buffer.read_string)
       end
 
+      unless topic_config.empty?
```

Comment:
Nitpick: won't the `.each` be a no-op if topic_config is `empty?`

Reply:
Ah! yeah that makes sense. After looking at this a little more I'm going to replace this with a guard for nil.